plugins/redis/lib_redis.ecllib

```diff
@@ -0,0 +1,87 @@
+  SetReal(CONST VARSTRING options, CONST VARSTRING key, REAL value, UNSIGNED database = 0, UNSIGNED4 expire = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRSetReal';
+  SetInteger(CONST VARSTRING options, CONST VARSTRING key, INTEGER value, UNSIGNED database = 0, UNSIGNED4 expire = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRSetInt';
+  SetUnsigned(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED value, UNSIGNED database = 0, UNSIGNED4 expire = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRSetUInt';
+  SetData(CONST VARSTRING options, CONST VARSTRING key, CONST DATA value, UNSIGNED database = 0, UNSIGNED4 expire = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRSetData';
+
+  INTEGER8 GetInteger(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetInt8';
+  UNSIGNED8 GetUnsigned(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetUint8';
+  STRING GetString(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetStr';
+  UNICODE GetUnicode(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetUChar';
+  UTF8 GetUtf8(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetUtf8';
+  BOOLEAN GetBoolean(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetBool';
+  REAL GetReal(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetDouble';
+  DATA GetData(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='SyncRGetData';
+
+  BOOLEAN Exists(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='RExist';
+  FlushDB(CONST VARSTRING options, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='RClear';
+  Del(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='RDel';
+  Persist(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='RPersist';
+  Expire(CONST VARSTRING options, CONST VARSTRING key, UNSIGNED4 expire, UNSIGNED database = 0) : cpp,once,context,namespace='RedisPlugin',entrypoint='RExpire';
+  INTEGER DBSize(CONST VARSTRING options, UNSIGNED database = 0) : cpp,action,context,namespace='RedisPlugin',entrypoint='RDBSize';
+END;
+
+export async := SERVICE : plugin('redis')
```

Comment:
can you add ,namespace('RedisPlugin') to the service and remove from each function?